src/custom-element.js

```diff
@@ -1076,14 +1086,16 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
     // 3. The element is too small or has not yet been measured;
     // 4. The element has already been laid out;
     // 5. The element is a `placeholder` or a `fallback`;
-    // 6. The element's layout is not a size-defining layout.
+    // 6. The element's layout is not a size-defining layout;
+    // 7. The element has loading error.
     if (this.loadingDisabled_ === undefined) {
       this.loadingDisabled_ = this.hasAttribute('noloading');
     }
     if (this.loadingDisabled_ || !isLoadingAllowed(this.tagName) ||
         this.layoutWidth_ < MIN_WIDTH_FOR_LOADING_ ||
         this.layoutCount_ > 0 ||
-        isInternalOrServiceNode(this) || !isLayoutSizeDefined(this.layout_)) {
+        isInternalOrServiceNode(this) || !isLayoutSizeDefined(this.layout_) ||
+        this.hadLoadingError_) {
```

Comment:
Put the `hadLoadingError_` check inside `isLoadingEnabled_` function, since we already check `isLoadingEnabled` before add loading dots. 
Also it's reasonable to disable loading when having error.